templates/zerver/help/user-groups.md

```diff
@@ -32,7 +29,33 @@ any user group they are a part of. Guests cannot modify user groups.
 
 {end_tabs}
 
+### Configure who can create and manage user groups
+
+{!admin-only.md!}
+
+By default, [all members](/help/roles-and-permissions) in a Zulip
+organization can create user groups and manage user groups that they
+are a member of. However, you can restrict that ability to specific
+[roles](/help/roles-and-permissions).
+
+Note that even when given permissions, members can only modify a user
+group if they are a member of it.
+
```

Comment:
Hm, this part is a bit confusing, since we're using "members" in two different ways. What is the actual rule, exactly? Do you have to be an admin to modify a group you aren't part of? What about moderators?

Reply:
Yeah, I simplified that on my last pass, but it didn't get clearer. The rule is that full and new members are limited to user groups they are a member of, but administrators and moderators can modify any user group in the org.